resources/sdk/purecloudgo/templates/apiclient.mustache

```diff
@@ -106,20 +147,8 @@ func (c *APIClient) CallAPI(path string, method string,
 	fileBytes []byte,
 	pathName string) (*APIResponse, error) {
 
-	// Build request URL w/query params
 	urlString := path + "?"
-	if queryParams != nil {
```

Comment:
looks queryparams logic is removed in the final method. believe we should include it